src/open_producten/producttypen/tests/api/test_link.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework.test import APIClient
+
+from open_producten.producttypen.models import Link, ProductType
+from open_producten.utils.tests.cases import BaseApiTestCase
+from open_producten.utils.tests.helpers import model_to_dict_with_id
+
+from ..factories import LinkFactory, ProductTypeFactory
+
+
+def link_to_dict(link):
+    link_dict = model_to_dict_with_id(link, exclude=["product_type"])
+    link_dict["product_type_id"] = link.product_type.id
+
+    return link_dict
+
+
+class TestProductTypeLink(BaseApiTestCase):
+
+    def setUp(self):
+        super().setUp()
+        product_type = ProductTypeFactory.create()
+        self.data = {
+            "naam": "test link",
+            "url": "https://www.google.com",
+            "product_type_id": product_type.id,
+        }
+        self.path = "/api/v1/links/"
```

Comment:
I think you should use reverse here:

```suggestion
        self.path = reverse("link-list")
```

And `reverse("link-detail")`